PriceCalculatorsKataTests/ProductTest.cs

```diff
@@ -0,0 +1,55 @@
+using NUnit.Framework;
+using PriceCalculatorKata;
+namespace PriceCalculatorsKataTests
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void productPricePercision()
```

Comment:
I know you wrote this before diving into the unit test courses, so I expect you'll start choosing better names for the testing methods. read more about unit test naming conventions.